style.css

```diff
@@ -0,0 +1,236 @@
+  background-position-x: center;
+  background-size: cover;
+  background-repeat: no-repeat;
+  height: 90vh;
+}
+
+.navbar {
+  border-bottom-left-radius: 0.5rem;
+  border-top-left-radius: 0.5rem;
+  border-bottom-right-radius: 0.5rem;
+  border-top-right-radius: 0.5rem;
+  max-width: 65%;
+  margin: auto auto;
+  background: linear-gradient(rgb(109, 108, 108) 35%, rgb(44, 43, 43) 50%, rgb(48, 43, 43) 5%);
+  position: relative;
+  top: 3rem;
+}
+
+.navbar-nav {
+  list-style-type: none;
+  padding: 0;
+  margin: 0;
+  overflow: hidden;
+}
+
+.nav-item {
+  float: left;
+}
+
+.apple-icon {
+  display: block;
+  text-align: center;
+  padding: 0.5rem 2rem;
+  font-size: 1.3rem;
+  color: rgb(175, 170, 170);
+  text-decoration: none;
+}
+
+.nav-link {
+  display: block;
+  color: white;
+  text-align: center;
+  padding: 0.5rem 1.2rem;
+  text-decoration: none;
+}
+
+#search {
+  padding-right: 0;
+}
+
+.border-right {
+  border-right: 1px solid rgb(44, 43, 43);
+}
+
+.border-right:not(:first-child):hover {
+  background: rgb(44, 43, 43);
+}
+
+input:focus {
+  outline: none;
+  background-color: #fff;
+}
+
+input[type="text"] {
+  border: 1px solid rgb(190, 185, 185);
+  border-radius: 6px;
+  width: 90%;
+  height: 1.2rem;
+  color: rgb(48, 44, 44);
+}
+
+.fa-search {
+  position: absolute;
+  right: 185px;
+  top: 0.9rem;
+  font-size: 0.8rem;
+  color: black;
+}
+
+.page-title {
+  font-family: "Montserrat", sans-serif;
+  font-size: 3.2rem;
+  color: #fff;
+}
+
+.text-div {
+  margin-top: 2rem;
+  line-height: 2rem;
+  font-size: 1.5rem;
+}
+
+.descrip-text {
+  font-family: "Quicksand", sans-serif;
+}
+
+.sm-descrip-text a {
+  font-family: "Quicksand", sans-serif;
+  font-family: "Dosis", sans-serif;
+  text-decoration: none;
+  color: #fff;
+}
+
+.descrip-text,
+.sm-descrip-text {
+  color: #fff;
+}
+
+.footer-link a {
+  text-decoration: none;
+  color: rgb(9, 156, 241);
+}
+
+.footer-nav a {
+  color: rgb(9, 156, 241);
+}
+
+.footer-nav li a {
+  display: block;
+  text-decoration: none;
+  padding: 0 0.4rem;
+}
+
+.sm-descrip-text a:hover {
+  text-decoration: underline;
+}
+
+.main-page-text {
+  position: relative;
+  top: 8rem;
+  text-align: center;
+}
+
+.img-container {
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-start;
+  height: 13rem;
+}
+
+.apple-img-1 {
+  background-image: url("./images/image1.jpg");
+  background-size: cover cover;
+  background-repeat: no-repeat;
+  width: 25%;
+  height: 100%;
+}
+
+.apple-img-2 {
+  background-image: url("./images/image2.jpg");
+  background-size: cover cover;
+  background-repeat: no-repeat;
+  width: 25%;
+  height: 100%;
+}
+
+.apple-img-3 {
+  background-image: url("images/image3.jpg");
+  background-size: cover cover;
+  background-repeat: no-repeat;
+  width: 25%;
+  height: 100%;
+}
+
+.apple-img-4 {
+  background-image: url("images/image4.jpg");
+  background-size: cover cover;
+  background-repeat: no-repeat;
+  width: 25%;
+  height: 100%;
+}
```

Comment:
This repetition can be refactored and `cover cover` is not a CSS property.

Reply:
Hello trekab

Thank you for your review. It is much appreciated and very helpful. I have done all the changes, however, I cannot give my `.apple-image-1, 2, & 3` the same rule as they have different background images. I am open to any suggestions and kind regards.